### DISCUSSION TITLE
Tag input fields

### DISCUSSION THREAD

**Discussion by ErikvdVen** [tabler/tabler#1563] [Ideas]
Tabler already contains those beautiful badges: https://preview.tabler.io/badges.html
Would it be possible to use them as tags as well? In an input field? 

So it would become easy to actually add tags to a form in an intuitive way? 
Or perhaps this already exists, but I wasn't able to find any in the past few hours.

So something like this, but within tabler:
https://bootstrap-tagsinput.github.io/bootstrap-tagsinput/examples/

**Comment by JobberRT** [comment]
See this closed issue: #1514 

It should be available soon, thanks to every contributor!
